README.md

```diff
@@ -54,6 +54,27 @@ mAwesomeValidation.addValidation(activity, R.id.edt_password, regexPassword, R.s
 // to validate a confirmation field (don't validate any rule other than confirmation on confirmation field)
 mAwesomeValidation.addValidation(activity, R.id.edt_password_confirmation, R.id.edt_password, R.string.err_password_confirmation);
 
+// to validate using your own custom validator function
+mAwesomeValidation.addValidation(activity, R.id.edt_birth, new CustomValidation() {
+    @Override
+    public boolean compare(String input) {
+        try {
+            new SimpleDateFormat("dd/MM/yyyy", Locale.US).parse(input);
+            return true;
+        } catch (Exception e) { return false;}
+        return false;
+    }
+}, R.string.err_birth);
+// or
+mAwesomeValidation.addValidation(editText, "Error info", new CustomValidation() {
```

Comment:
I am aware of that, It's actually an override to the API design because for lots of developers it is more convenient. It is also a better code style. If you think it's better to not have it, I will remove it!

Reply:
I guess you meant overload, since `AwesomeValidation` doesn't extend or implement any other class or interface, it can not override any method.  Yes, please put the `"Error info"` to the end to keep the consistency.  Thanks.